package/python-kiwi-spec-template

```diff
@@ -294,6 +294,9 @@ Requires:       curl
 %if 0%{?debian} || 0%{?ubuntu}
 Requires:       xz-utils
 Requires:       dmsetup
+Requires:       btrfs-tools
+Requires:       gdisk
+Requires:       dracut-network
```

Comment:
`dracut-network` is also separate in Fedora/CentOS, since the upstream dracut spec file recommends it that way.

Reply:
oh, ok yes we should have a Fedora iso test build as well, the existing virtual disk image build does not show this. I'll adapt the spec change